### PR TITLE
fix(salsah2xml): some housekeeping

### PR DESCRIPTION
**Pull request by musicEnfanthen**
This PR provides some housekeeping for the salsah2xml.py script. It does not change any logic, but adds and corrects typings, fixes some typos, removes unused imports and redundant lines, and adds a requirements.txt file.

Maybe you can have a look @Vijeinath ?

### CODE REVIEW

File: salsah2xml.py [hunk 1]
```diff
@@ -1104,7 +1104,7 @@ def program(args):
     parser.add_argument("-S", "--start", type=int, help="Start at record with given number")
     parser.add_argument("-F", "--folder", default="-", help="Output folder")
     parser.add_argument("-r", "--resptrs_file", help="list of resptrs targets")
```
Comment (Vijeinath) [inline salsah2xml.py:1106]: If you want to follow the help text rules of the other arguments, you can also start with the capital letter

**Review by Vijeinath — APPROVED** [review]
I'm happy with it

File: salsah2xml.py (hunk 1 above)
Comment (musicEnfanthen) [reply salsah2xml.py]: Changed it.

**Comment by musicEnfanthen** [comment]
Thank you. So ready to be merged.

**Comment by musicEnfanthen** [comment]
Thank you!

**Pull request merged**